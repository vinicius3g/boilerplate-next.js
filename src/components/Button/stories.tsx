import Button from '.'
import { Story, Meta } from '@storybook/react/types-6-0'

export default {
  title: 'Button',
  component: Button
} as Meta

export const Basic: Story = () => <Button />
