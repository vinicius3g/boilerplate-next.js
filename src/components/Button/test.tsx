import { render, screen } from '@testing-library/react'

import Button from '.'

describe('<Button />', () => {
  it('should render the hading', () => {
    const { container } = render(<Button />)

    expect(screen.getByRole('heading', { name: /Button/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
