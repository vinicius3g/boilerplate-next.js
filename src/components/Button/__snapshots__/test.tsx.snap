// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the hading 1`] = `
<main
  class=""
>
  <h1>
    Button
  </h1>
</main>
`;
